### PR TITLE
Artistic Style Transfer using CycleGAN

### PR DESCRIPTION
**Pull request by BlueBlaze6335**
#68 Generating Van Gogh style pictures from random normal pictures !!

### CODE REVIEW

**Comment by BlueBlaze6335** [comment]
@smaranjitghose  dada please check the  pr !!

File: Artistic Style Transfer with CycleGAN/Readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Artistic Style Transfer With CycleGAN :
+
+* In this [notebook](https://colab.research.google.com/drive/19t6m05nv7gnVdRZMkD-XZnaWUIsKcKZF?usp=sharing) we will explore the Artistry of the GANs , that is we will create Van Gogh Style pictures , by learning from his paintings and implement his ***Artistic*** style on Normal images.
```
Comment (smaranjitghose) [inline Artistic Style Transfer with CycleGAN/Readme.md]: Not Required.

Just put a section in the readme as Style Transfer and have three columns: Content, Style and Output

All images should be in assets folder

File: Artistic Style Transfer with CycleGAN/Readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+* Now what is GAN ? GANs are like a pair of a teacher and a student , in this case , the teacher is Van Gogh and the student is someone who knows painting. So , what the student does is , he pics a random picture from the internet , and tries to paint it but in Van Gogh's style. When he finishes the painting , Van Gogh judges the painting , and tells him if he did it right or wrong. Based on his assesment , the student improves their work.
+
+* Then what is ***CycleGAN*** ??? CycleGAN is like , the student becomes the teacher for once and Van Gogh becomes the student , turn by turn. The student trying to copy Van Gogh , and Van Gogh trying to copy the students style of painting the normla pictures. Now you must be thinking , why does Van Gogh need to know how to make normal paintings , isnt his paintings **Awesome** ?? Well , Van Gogh is a very good teacher. He tries to learn the technique , so that he can teach his student better and judge him more precisely , only to make his student excell the ***Artistic Style***.
+
```
Comment (smaranjitghose) [inline Artistic Style Transfer with CycleGAN/Readme.md]: Add a script that takes in a content image, style image (paths using argparse) and generates the output using the cyclegan used in the notebook above. Use tqdm for colorful progress bars

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by BlueBlaze6335** [comment]
I am having a problem in pushing the changed files , thats why i am closing this and i will add a new pr !! Sorry for the inconvenience @smaranjitghose .

**Pull request closed without merging**